src/nvhttp.cpp

```diff
@@ -267,12 +268,32 @@ namespace nvhttp {
 
     launch_session.host_audio = host_audio;
     launch_session.gcm_key = util::from_hex<crypto::aes_t>(get_arg(args, "rikey"), true);
+    std::stringstream mode = std::stringstream(get_arg(args,"mode"));
+    //Split mode by the char "x", to populate width/height/fps
+    int x = 0;
+    std::string segment;
+    while(std::getline(mode, segment, 'x'))
+    {
+      if(x == 0)launch_session.width = stoi(segment);
+      if(x == 1)launch_session.height = stoi(segment);
+      if(x == 2)launch_session.fps = stoi(segment);
```

Comment:
Please adjust spacing around `if` and `while` to be consistent with the rest of the file.